src/_scss/pages/search/_toggleTempSearchPage.scss

```diff
@@ -0,0 +1,47 @@
+// this file can be deleted once the new advance search has been implemented
+// and the toggle between new and old search ui is no longer needed
+
+.temp-search-page {
+  display: grid;
+}
+
+.rounded-toggle__wrapper {
+  margin: rem(16) rem(24) rem(8) rem(24);
```

Comment:
For sizes 768-992 the margin-top needs to be 24, and remain 16 for sizes < 768

Reply:
Thank you for catching this. It has been updated.